gitops/binding_test.go

```diff
@@ -3,14 +3,13 @@ package gitops_test
 import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
```

Comment:
Why this change? It looks like fight of IDEs, where one moves it to top and other back to bottom of this section.